flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_FlightPlanPage.js

```diff
@@ -738,6 +738,20 @@ class CDUFlightPlanPage {
         }
         const allowScroll = waypointsAndMarkers.length > 4;
         if (allowScroll) {//scroll only if there are more than 5 points
+            //Only need to call onAirport()if there are > waypoints
+        //since both origin and destination are already displayed on the same page if no
```

Comment:
I like comments such as these explaining a piece of code. However, it is not quite clear to me what the comments indicate. It would be great if you could clarify them a bit.

Also, we typically use the following comment format for comments:
```javascript
// This is a comment
```
over this:
```javascript
//This is a comment
```
I can see that this convention has not really been followed in some of the already existing code either, but I think sticking to the convention for new code makes sense.